buildSrc/build.gradle

```diff
@@ -2,7 +2,3 @@ repositories {
     mavenLocal()
     maven { url "https://plugins.gradle.org/m2/" }
 }
-
-dependencies {
-    implementation "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.4"
```

Comment:
Unrelated, but this plugin no longer exists and fails the build. And this lib has switched to GithubPackages for publishing anyways.